pkgs/tools/text/vale/default.nix

```diff
@@ -26,6 +34,16 @@ buildGoModule rec {
   # Tests require network access
   doCheck = false;
 
+  passthru.withStyles = selector: symlinkJoin {
+    name = "vale-with-styles-${vale.version}";
+    paths = [ vale ] ++ selector valeStyles;
+    nativeBuildInputs = [ makeBinaryWrapper ];
+    postBuild = ''
+      wrapProgram "$out/bin/vale" \
+        --set VALE_STYLES_PATH "$out/share/vale/styles/"
+    '';
+  };
+
   meta = with lib; {
     description = "A syntax-aware linter for prose built with speed and extensibility in mind";
```

Comment:
Maybe we could add a `longDescription` to vale hinting how to use vale with styles?

Reply:
Good idea, I added a small documentation